### DISCUSSION TITLE
[firebase_analytics] Ignore testlab devices

### DISCUSSION THREAD

**Discussion by ciriousjoker** [firebase/flutterfire#5630] [Feature request]
**Is your feature request related to a problem? Please describe.**
When distributing an app to the play store, Google automatically tests the app. This generates fake analytics.

**Describe the solution you'd like**
Please ignore testlab devices when sending analytics data.

**Additional context**
Here's how this could be implemented. Check for the value of the system setting "firebase.test.lab".

More details:
https://stackoverflow.com/questions/39404078/how-to-prevent-play-store-testing-from-affecting-firebase-analytics

**Comment by RicardoRB** [comment]
How is the progress of this? It would be great to have it. My implementation doesn't work.

**Comment by RicardoRB** [comment]
Anyone has a way to resolve this? @Ehesp  @kirstywilliams  @Salakar @helenaford
